packages/sdk-demo/src/components/Layout/index.tsx

```diff
@@ -22,15 +25,32 @@ type DefaultLayoutProps = {
 export const DefaultLayout = ({ children, siderProps }: DefaultLayoutProps) => {
   const location = useLocation();
   const { data: user } = useEntityUserByAuthToken();
+  const [config, setConfig] = useState<ConfigSchema | null>(null);
   const [pagePadding, setPagePadding] = useState(4);
 
-  React.useEffect(() => {
+  useEffect(() => {
+    const fetchConfig = async () => {
```

Comment:
It's time, to use the full power is Suspense! We already used this [approach](https://github.com/team-monite/monite-sdk/blob/0ccc9f84afc5a60351dcc90186b98746c68bb7e9/packages/sdk-react/src/core/context/I18nLocaleProvider.tsx#L97) to compile translation messages in runtime.

Brief implementation:

```tsx
import React, { lazy, memo, Suspense } from 'react';

const ConfigProvider = memo(lazy(async () => {
  const config = await getConfig();
  return {
    default: function ConfigProviderLoader({ children }: { children: React.ReactNode }) {
      return (
        <ConfigContext.Provider value={config}>
          {children}
        </ConfigContext.Provider>
      )
    }
  }
}));

const 

```

Reply:
the config context created